### PR TITLE
- New Api Project

### PR DESCRIPTION
**Pull request by coffeelake**
- HealthController.cs - Added and returns the Health Tick
- .gitignore added, ignoring appsettings.json and appsettings.Development.json

### CODE REVIEW

File: src/Api/.gitignore
Comment (coffeelake) [inline src/Api/.gitignore:1]: this should use root gitignore file instead

File: src/Api/Dockerfile
Comment (coffeelake) [inline src/Api/Dockerfile:1]: should be renamed to mach api and put in root beside the other docker file

File: src/Api/Program.cs
Comment (coffeelake) [inline src/Api/Program.cs:1]: break out a Stratup file like web project

**Review by coffeelake — COMMENTED** [review]
new review

File: src/Api/.gitignore
Comment (sorrz) [reply src/Api/.gitignore:1]: Done

**Comment by sorrz** [comment]
Closed due to merge conflicts

**Pull request closed without merging**